frontend/src/fixtures/personalSchedulesFixtures.js

```diff
@@ -0,0 +1,30 @@
+const personalSchedulesFixtures = {
+    onePersonalSchedule: {
+        "id": 1,
+        "name": "Aragorn",
+        "description": "Sword fighting in the morning, politics in the afternoon",
+        "quarter": "Spring"
```

Comment:
Quarter should be stored in the format YYYYQ.

Please replace this in the fixtures with a YYYYQ format quarter.